HW_1/task4.py

```diff
@@ -0,0 +1,10 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# cколько журавликов сделал каждый ребенок, если известно,
+# что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# Если мальчики сделали по x журавликов, то вместе они сделают 2x, а девочка сделала в два раза больше их суммы
+# Из этого следует, что
+# S = 2x + (2x*2)
+# S = 6x
+s = int(input('Введите общее количество журавликов: '))
+print('Петя сделал', s//6, ', Катя сделала', (s//6)*4, ', Сережа сделал', s//6,)
```

Comment:
Во первых, не помешало бы проверить входные данные на кратность 6. Во-вторых, у операции умножения и деления одинаковый приоритет, то есть скобки не нужны. В-третьих, у вас повторяется одно и то же действие. Для того, чтобы этого не происходило, используют переменные.
```suggestion
print('Петя сделал', var := s // 6, ', Катя сделала', var * 4, ', Сережа сделал', var)
```